src/scripts/main.js

```diff
@@ -559,6 +560,17 @@
         sendMessageToServer(db_url, message);
     }
 
+    // Add bonus journal entry stuff to the hunt_details
+    function addMoreDetails(hunt) {
+        const new_details = {};
+        if ('more_details' in hunt) {
+            hunt.more_details.forEach(detail => {
+                new_details[detail] = true;
```

Comment:
This approach will work so long as all hunt_details we want to add are booleans. Not sure how robust that assumption is.

That said, this function is effectively the same as `Array#reduce` - take an array and accumulate all its values in the same way.